lib/view.js

```diff
@@ -267,3 +267,43 @@ export function reactVirtualizedToReactVirtualized(view) {
     , columnWidth
   }
 }
+
+/**
+  * Ensure dates and times in resources are in ISO format.
+  * @param {resource}
+  * @return {resource} with formatted dates and times
+  */
+export function normalizeDateAndTime(resource) {
```

Comment:
comparing this and one in the `dist/views` - they are identical right? why are not we importing one of them? Not getting the idea of having it in two different places